executor/index_lookup_join_test.go

```diff
@@ -394,8 +470,11 @@ PARTITIONS 1`)
 	tk.MustQuery("select /* +INL_JOIN(t1,t2) */ t1.id, t1.pc from t1 where id in ( select prefiller from t2 where t2.postfiller = 1 )").Check(testkit.Rows())
 }
 
-func (s *testSuite5) TestIssue27893(c *C) {
-	tk := testkit.NewTestKit(c, s.store)
+func TestIssue27893(t *testing.T) {
```

Comment:
please move to `index_lookup_join_serial_test.go`

Reply:
add parallel